### ISSUE TITLE
strangeness with warning and confirmation messages on RT upload

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#534]
tested in DEV with Firefox with Profiles containing only one RT each

1) attempt to upload existing template friday5, get warning as expected "friday5 already exists. Do you want to overwrite these resource templates?", choose "Yes, overwrite", get blue message as expected "Attempting to update, Updated". Everything works so far.
2) try to upload another RT which is called friday5, but which has some internal changes (resource and property label). get message "friday5 already exists, friday5 already exists" as if the message from #1 didn't get cleared. choose "yes overwrite", blue message is "Attempting to update , Updated , Attempting to update , Unexpected response (500)! , Updated" again as if the message from #1 is still there (and don't know what the 500 error is but the revised RT was successfully uploaded and the old one overwritten). (I checked that the Resource label was updated as per the new version, and opened the RT to make sure the new property label was showing and both those things worked fine.)
3) now I close the blue message (with the little x) to see if the next time I do it, I will get a fresh confirmation message. but now the blue message doesn't appear at al when I try to upload a new or revised resource template (though the new or revised does seem to upload successfully). If I click away from that tab and then back, the blue message comes back the next time I upload, and the blue message is "refreshed", i.e., it doesn't have any of the old confirmation messages.

basically if I keep trying to upload existing Resource Templates, the overwrite warning keeps getting longer and longer, like this (this is on the 3rd time uploading a saturday2 template)
<img width="648" alt="Screen Shot 2019-05-18 at 8 56 33 AM" src="https://user-images.githubusercontent.com/23069165/57972178-cff5a080-794b-11e9-823e-69ada4625439.png">

and the blue confirm message keeps getting longer and longer, like:
<img width="1125" alt="Screen Shot 2019-05-18 at 8 56 43 AM" src="https://user-images.githubusercontent.com/23069165/57972209-2531b200-794c-11e9-9af9-80daddf975c2.png">
